app/dao/service_permissions_dao.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import exc
+
+from app import db
+from app.dao.dao_utils import transactional
+from app.models import Service, ServicePermission, SERVICE_PERMISSION_TYPES
+
+
+def dao_fetch_service_permissions(service_id):
+    return ServicePermission.query.filter(
+        ServicePermission.service_id == service_id).all()
+
+
+@transactional
+def dao_add_and_commit_service_permission(service_id, permission):
```

Comment:
Can we rename this to `dao_create_service_permission`? It's implicit in that we'll add and commit a service permission.